app/src/main/kotlin/com/wire/android/navigation/NavigationManager.kt

```diff
@@ -37,8 +40,7 @@ class NavigationManager {
         // so the navigate command goes nowhere.
         // To avoid such lose we'll wait till _navigateState Flow is subscribed and emit command into it only after that.
         _navigateState.subscriptionCount
-            .filter { it > 0 }
-            .take(1)
+            .dropWhile { it <= 0 }
```

Comment:
I think it still needs to `.take(1)`, I probably wrongly selected both lines 🙈 But without `.take(1)` it will emit the command every time subscription count changes, right? 😅